packages/core/src/browser/keybinding.ts

```diff
@@ -533,6 +533,36 @@ export class KeybindingRegistry {
         return input;
     }
 
+    registerEventListeners(win: Window): Disposable {
+        /* vvv HOTFIX begin vvv
+        *
+        * This is a hotfix against issues eclipse/theia#6459 and gitpod-io/gitpod#875 .
+        * It should be reverted after Theia was updated to the newer Monaco.
+        */
```

Comment:
It appears that the issues mentioned here have been closed. Do we still need this?

Reply:
I don't know, but the issues were closed because this fix was added. See https://github.com/eclipse-theia/theia/pull/6673/files. If we want a "real" fix, the proper way would be to reopen the issue. But I feel that would be different work from what this PR adresses.